spec/features/checking_in_spec.rb

```diff
@@ -14,24 +11,20 @@
     fill_in 'Day', with: '1 January, 2016'
     fill_in 'pick-a-time', with: '4:30 PM'
     select 'Start Shift', from: 'Action'
-    find(:css, '#check_in_form_signature', visible: false).set 'my signature'
+    find('#check_in_form_signature', visible: false).set 'my signature'
     click_button 'Save'
 
-    expect(Shift.count).to eq(shift_count + 1)
-    expect(ShiftEvent.count).to eq(shift_event_count + 1)
-    expect(Volunteer.count).to eq(volunteer_count + 1)
-
-    shift_event = ShiftEvent.first
-    expect(shift_event.action).to eq('start_shift')
-    expect(shift_event.occurred_at).to eq(Time.zone.local(2016, 1, 1, 16, 30))
+    expect(page).to have_content 'Check in has been saved.'
```

Comment:
I was thinking that if we have the form unit tests verifying that the data is saved when the form is successfully saved, then  in the feature spec, when the flash confirms that the form was saved successfully, we can safely assume that the data was persisted.